dosomething-dev/main.tf

```diff
@@ -68,20 +64,18 @@ module "bertly" {
 }
 
 module "fastly-frontend" {
-  source = "./fastly-frontend"
-
-  phoenix_name    = module.phoenix.name
-  phoenix_backend = module.phoenix.backend
+  source = "../components/fastly_frontend"
+  name   = "Terraform: Frontend (Development)"
 
+  application            = module.phoenix
   papertrail_destination = var.papertrail_destination_fastly
-  papertrail_log_format  = local.papertrail_log_format
 }
 
 module "fastly-backend" {
   source = "../components/fastly_backends"
-  name   = "Fastly: Backends (Development)"
+  name   = "Terraform: Backends (Development)"
```

Comment:
I'd accidentally prefixed these `Fastly:` in my previous PR... which, duh! Of course they're Fastly! I'm not sure how important it is anymore, but the reason I'd prefixed these with `Terraform:` originally was to indicate to anyone logged into the web console that they should make edits there (not via the GUI)!

Reply:
Oh smart prefixing there!